examples/remarketing/upload_call_conversion.rb

```diff
@@ -0,0 +1,160 @@
+# limitations under the License.
+#
+# This example imports offline call conversion values for calls related to the
+# ads in your account.
+# To set up a conversion action, run the add_conversion_action.rb example.
+
+require 'optparse'
+require 'google/ads/google_ads'
+require 'date'
+
+def upload_call_conversion(customer_id, conversion_action_id, caller_id,
+  call_start_date_time, conversion_date_time, conversion_value)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  # Create a call conversion by specifying currency as USD.
+  call_conversion = client.resource.call_conversion do |c|
+    c.conversion_action = client.path.conversion_action(
+      customer_id, conversion_action_id)
+    c.caller_id = caller_id
+    c.call_start_date_time = call_start_date_time
+    c.conversion_date_time = conversion_date_time
+    c.conversion_value = conversion_value
+    c.currency_code = "USD"
+  end
+
+  # Issues a request to upload the call conversion.
+  response = client.service.conversion_upload.upload_call_conversions(
+    customer_id: customer_id,
+    conversions: [call_conversion],
+    partial_failure: true
+  )
+
+  # Prints errors if any partial failure error is returned.
+  if response.partial_failure_error
+    failures = client.decode_partial_failure_error(response.partial_failure_error)
+    failures.each do |failure|
+      failure.errors.each do |error|
+        human_readable_error_path = error
+          .location
+          .field_path_elements
+          .map { |location_info|
+            if location_info.index
+              "#{location_info.field_name}[#{location_info.index}]"
+            else
+              "#{location_info.field_name}"
+            end
+          }.join(" > ")
+
+        errmsg =  "error occured while adding operations " \
+          "#{human_readable_error_path}" \
+          " with value: #{error.trigger.string_value}" \
+          " because #{error.message.downcase}"
+        puts errmsg
+      end
+    end
+  else
+    # Print the result if valid.
+    uploaded_call_conversion = response.results.first
+    puts "Uploaded call conversion that occurred at " \
+      "#{uploaded_call_conversion.call_start_date_time} " \
+      "for caller ID " \
+      "#{uploaded_call_conversion.caller_id} " \
+      "to the conversion action with resource name " \
+      "#{uploaded_call_conversion.conversion_action}"
+  end
+end
+
+if __FILE__ == $0
+  options = {}
+  # The following parameter(s) should be provided to run the example. You can
+  # either specify these by changing the INSERT_XXX_ID_HERE values below, or on
+  # the command line.
+  #
+  # Parameters passed on the command line will override any parameters set in
+  # code.
+  #
+  # Running the example with -h will print the command line usage.
+  options[:customer_id] = 'INSERT_CUSTOMER_ID_HERE'
```

Comment:
Missing other default values.

Reply:
per above, only CUSTOMER_ID is needed here.